.travis.yml

```diff
@@ -3,12 +3,6 @@ node_js:
 - 6
 - 4
 
-matrix:
-  fast_finish: true
```

Comment:
Also, OSX builds are **always** longer than Linux builds, so it would alway fast finish...